content/upgrades/index.md

```diff
@@ -0,0 +1,436 @@
+# Best Practices for Cluster Upgrades
+
+This guide shows cluster administrators how to plan and execute their Amazon EKS upgrade strategy. It also describes how to upgrade self-managed nodes, managed node groups, and Fargate nodes. It does not include guidance on EKS Anywhere, self-managed Kubernetes, AWS Outposts, or AWS Local Zones. 
+
+## Overview
+
+A Kubernetes version encompasses both the control plane and the data plane. To ensure smooth operation, both the control plane and the data plane should run the same [Kubernetes minor version, such as 1.24](https://kubernetes.io/releases/version-skew-policy/#supported-versions). 
+
+* **Control plane** — The control plane version is defined by the Kubernetes API server. In EKS clusters, this is managed by AWS. Upgrades to the control plane version are started using the AWS API. 
+* **Data plane** — The data plane version references the version of the Kubelet running on your nodes. Different nodes in the same cluster may have different versions. See the version of all nodes with `kubectl get nodes`. 
+
+## Keep your cluster updated
+
+Keeping up with Kubernetes releases is a critical component of the shared responsibility model when adopting EKS and Kubernetes. 
+
+Past a certain point (usually 1 year), the Kubernetes community stops releasing bug and CVE patches. Additionally, the Kubernetes project does not encourage CVE submission for deprecated versions. This means that vulnerabilities specific to an older version of Kubernetes may not even be reported, leaving you exposed with no notice and no remediation options in the case of a vulnerability.
```

Comment:
> /s/bug/bug fixes.

done, ty.

> Is this documented somewhere?

The kubernetes security vulernability site links to supported versions. I'd prefer to stay with this paragraph as written because it's the standardlized/approved language from Vipin we use in other contexts.